app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,83 +1,165 @@
+  <div class="top-detail" >
+    <%= form_for :order, html: {
+      id: 'add-to-cart-form',
+      class: 'add-to-cart-form',
+      'data-product-summary': @product_summary.to_json,
+      'data-variants': product_variants_matrix(is_product_available_in_currency),
+      'data-variant-change-trigger-identifier': variant_change_identifier
+      } do |f| %>
+        <div class="product-options-header d-flex align-items-center justify-content-between" id="inside-product-cart-form" data-hook="inside_product_cart_form">
+          <div class="left">Choose your <strong>option:</strong></div>
+          <div class="right">
+            <div class="text-center text-md-left text-uppercase " id="product_variant_sku">
+                <%= default_variant.sku  %>
+            </div>
+          </div>
+        </div>
+        <ul class="size-availability" id="product-variants">
+          <% if @product.variants_and_option_values(current_currency).any? %>
+            <% @product.variants_and_option_values(current_currency).each_with_index do |variant, index| %>  
+              <li class="d-flex flex-wrap align-items-center justify-content-between  product-ov-js <%= index == 0 ? 'active' : '' %>"
+                data-varient-id="<%= variant.id %>" data-varient-sku="<%= variant.sku %>"> 
+              <% option_value = variant.size_options %>
+                <div class="left  add-to-cart-form-price price selling"  id="product-price"  data-hook="product_price">
+                <%= variant.price_in(current_currency).money %><span>/ $6.29 per oz.</span></div>
+                <div class="right"> 
+                  <%= option_value&.presentation %>
+                  <%= radio_button_tag "variant_id",  variant.id, index == 0 ? true : false,
+                          'data-price' => variant.price_in(current_currency).money,
+                          'data-in-stock' => variant.can_supply?,
+                          'data-backordered' => variant.backordered?,
+                          style: "display:none"
+                      %>
+            <% end%>
+              </div>
+          <% else %>
+            <%= hidden_field_tag "variant_id", @product.master.id %>
+              </li>
+          <% end %>
+        </ul>
+        <div class="d-flex justify-content-end mb-5">
+          <button type="button" class="btn btn-white expand-list">More Options</button>
+        </div>
+        <% if is_product_available_in_currency && @product.can_supply? %>
+          <%= hidden_field_tag "variant_id", default_variant.id %>
+            <div class="add-to-cart-wrap mb-30">
+              <label>Quantity:</label>
+              <div class="d-flex flex-wrap align-items-center">
+                <%= render 'spree/shared/quantity_select', input_name: :quantity %>
+                  <%= button_tag class: 'btn btn-primary add-to-cart-button', id: 'add-to-cart-button', type: :submit do %>
+                    <%= Spree.t(:add_to_cart) %>
```

Comment:
Total order amount is missing from cart label as per template.
<img width="194" alt="Screen Shot 2021-12-09 at 1 32 06 AM" src="https://user-images.githubusercontent.com/77246192/145279817-ebc24a90-ad0b-4e4a-8e0a-2d90e5ca2fc8.png">